tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -119,35 +150,2072 @@ def test_get_metrics_and_expectations(
+                            "domain_kwargs": {"column": "vendor_id"},
+                            "metric_value_kwargs": None,
+                            "metric_dependencies": None,
+                        },
+                        "num_batches": 36,
+                    }
+                },
+                "expectation_type": "expect_column_unique_value_count_to_be_between",
                 "kwargs": {
-                    "min_value": 10000,
-                    "max_value": 10000,
+                    "strict_max": False,
+                    "max_value": 3,
+                    "strict_min": False,
+                    "column": "vendor_id",
+                    "min_value": 2,
                 },
-                "expectation_type": "expect_table_row_count_to_be_between",
+            }
+        ),
+        ExpectationConfiguration(
+            **{
                 "meta": {
                     "profiler_details": {
                         "metric_configuration": {
-                            "metric_name": "table.row_count",
-                            "domain_kwargs": {},
+                            "metric_name": "column.distinct_values.count",
+                            "domain_kwargs": {"column": "pickup_datetime"},
                             "metric_value_kwargs": None,
                             "metric_dependencies": None,
                         },
                         "num_batches": 36,
-                    },
+                    }
                 },
-            },
-        )
-    )
-
-    expected_expectation_configurations: List[ExpectationConfiguration] = [
-        expected_expectation_configuration,
+                "expectation_type": "expect_column_unique_value_count_to_be_between",
+                "kwargs": {
+                    "strict_max": False,
+                    "max_value": 9983,
+                    "strict_min": False,
+                    "column": "pickup_datetime",
+                    "min_value": 9944,
+                },
+            }
+        ),
+        ExpectationConfiguration(
+            **{
+                "meta": {
+                    "profiler_details": {
+                        "metric_configuration": {
+                            "metric_name": "column.distinct_values.count",
+                            "domain_kwargs": {"column": "dropoff_datetime"},
+                            "metric_value_kwargs": None,
+                            "metric_dependencies": None,
+                        },
+                        "num_batches": 36,
+                    }
+                },
+                "expectation_type": "expect_column_unique_value_count_to_be_between",
+                "kwargs": {
+                    "strict_max": False,
+                    "max_value": 9985,
+                    "strict_min": False,
+                    "column": "dropoff_datetime",
+                    "min_value": 9957,
+                },
+            }
+        ),
```

Comment:
Nitpick - Any reason why we don't just instantiate the object with keyword args instead of using **kwargs unpacking? I feel like there's no real need for a dict here but I've seen this pattern quite a bit.

Reply:
@cdkini Editing convenience. :-)